internal/manifest/manifest.go

```diff
@@ -387,3 +387,13 @@ func (m *Manifest) TDXValidateOpts() ([]*tdxvalidate.Options, error) {
 
 	return out, nil
 }
+
+// CoordinatorPolicyHash returns the hash of the coordinator policy.
+func (m *Manifest) CoordinatorPolicyHash() (HexString, error) {
+	for policyHash, policy := range m.Policies {
+		if policy.Role == RoleCoordinator {
+			return policyHash, nil
```

Comment:
The old function had a comment about multiple policies with coordinator role. Should we check that in manifest validation?

Reply:
added validation for it in https://github.com/edgelesssys/contrast/pull/1280/commits/2e1add95eaf68888753aa209592597820572c770